chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -10,36 +10,30 @@ UPGRADEDIR=/var/lib/crowbar/upgrade
 mkdir -p "`dirname "$LOGFILE"`"
 exec >>"$LOGFILE" 2>&1
 
+hostname=$1
+if [[ "$2" = "delete-ns" ]]; then
+    delete_ns="--ssh-delete-namespace"
+else
+    delete_ns=""
+fi
+
 echo "Executing $BASH_SOURCE"
 
 set -x
 
 mkdir -p $UPGRADEDIR
 
-if [[ -f $UPGRADEDIR/crowbar-router-migration-ok ]] ; then
```

Comment:
Why are we removing this part? If I understand it correcly, if the migration is successful, crowbar will remove the files. I looked for occurances of `crowbar-router-migration-failed` and `crowbar-router-migration-ok`, but found none, only this file creates them, then we remove them from crowbar. What is the purpose of these files?